database/migrations/2022_10_12_130606_create_login_histories_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('login_history', function (Blueprint $table) {
+            $table->id();
+            $table->integer('user_id');
+            $table->dateTime('enter_time');
+            $table->string('ip', 25);
```

Comment:
в ларавель есть хелпер метод для ip)
можно не исправлять (для общей сведения)
```suggestion
            $table->ipAddress('ip');
```